sys/androidmedia/gstamc.c

```diff
@@ -131,6 +132,18 @@ static struct
   jmethodID from_string;
 } uuid;
 
+static const struct
+{
+  gint id;
```

Comment:
this field is not used. And if it will be used, it doesn't make sence, because if you access the array, you already know the id.

The only use case for this field I can imagine is for a sanity check in a loop:

while (features_to_check[feature_idx].str) {
   if (features_to_check[feature_idx].id != feature_idx)
      g_abort (); // but this is also pretty useless, given that it's not a compile time check
}